### ISSUE TITLE
make it clear that you can specify the stat inside a geom() function

### ISSUE THREAD

**Issue by Thomas-Hirsch** [moj-analytical-services/ggplotTraining#5]
There needs to be another slide mentioning that geom functions have a stat parameter, which could be used as an alternative to the stat_ functions. Currently slide 26 mentions stats offhand, and 28 refers to using the ..name.. shortcut for referring to stats, but in the solutions slide 16 the solution uses geom_bar(stat="summary",fun.y="mean"). Both stat= and stat_summary/fun.y deserve more explanation.

**Comment by Thomas-Hirsch** [comment]
ggplot2 3.0.0 (https://github.com/tidyverse/ggplot2/releases/tag/v3.0.0) has a very relevant update:

> New syntax for calculated aesthetics. Instead of using aes(y = ..count..)
> you can (and should!) use aes(y = stat(count)). stat() is a real function
> with documentation which hopefully will make this part of ggplot2 less
> confusing (#2059).

I think it's worth incorporating this.

**Comment by MattFunnell** [comment]
Agreed. This is definitely a point that will need to be explained and emphasised by the presenter in sessions, as is tricky to understand.
I’ve added what will hopefully be some helpful slides to illustrate. Slides 26-28 (of the new branch) illustrate an example of switching stat and geom; and I’ve added some more text on what is now slide 31, to explain the new feature for specifying a stat when we just have one x variable.

Amended in branch oct_18_issues
